packages/lesswrong/server/notificationBatching.tsx

```diff
@@ -121,8 +135,13 @@ export const graphqlQueries = {
       const notifications = await Notifications.find(
         { _id: {$in: notificationIds} }
       ).fetch();
+
+      // Assume they are all of the same type
+      const notificationType = notifications[0].type;
```

Comment:
This was already assumed, I changed it to be a parameter that is passed through to make it more obvious that the list of notifications has this constraint